.env.example

```diff
@@ -0,0 +1,2 @@
+VITE_MIXPANEL_DEV_TOKEN=mixpanel_dev_token
+VITE_MIXPANEL_PROD_TOKEN=mixpanel_prod_token
```

Comment:
I understand that all `.env` files should not exist in GitHub history, so I recommend [removing these records](https://ebang.tistory.com/108).
<img width="928" alt="image" src="https://github.com/user-attachments/assets/a35d02b5-a34e-4582-9f48-4b64ee9aea57" />

Is there any reason that you namaing as `.env.exmaple` instead of `.env`?

Reply:
`.env.example` is just a example for `.env` file! 

Like you said `.env` files should not exists in GitHub but I think there should be examples of what variables are in the `.env` file, so I added `.env.example`

Values defined in .env.example are not valid.